tests/jballerina-unit-test/src/test/resources/test-src/types/never/never-type-runtime.bal

```diff
@@ -78,6 +78,39 @@ function test6() {
     panic error("Error value is:" + string `${b}`);
 }
 
+function testNeverWithAnyAndAnydataRuntime() {
+    map<never> a = {};
+    never[] b = [];
+
+    anydata m = a;
+    assertEquality(true, m is map<any>);
+    assertEquality(true, m is any);
```

Comment:
Shall we assign `a` to an `any` variable and specifically check `is anydata` also?

Reply:
Fixed in b9d9424 
Came across https://github.com/ballerina-platform/ballerina-lang/issues/30264